File: thomas.breigenzer/Homework 3/HelloWFA/HelloWFA/Form1.cs
using System;
using System.Windows.Forms;

namespace HelloWFA
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();
        }

        private void Button1Click(object sender, EventArgs e)
        {
            string nameEnteredInTextBox = textBox1.Text;
            if (nameEnteredInTextBox == null) throw new ArgumentNullException("sender");
            MessageBox.Show(string.Format("Hello, {0}", nameEnteredInTextBox));
        }
    }
}

File: Phil Melton/Homework 3/Form1.vb
Public Class Form1

    Private Sub Form1_Load(sender As Object, e As EventArgs) Handles MyBase.Load, MyBase.Click, Button1.Click, Button1.TextChanged

        MessageBox.Show(String.Format("Hello, {0}!", TextBox1.Text))

    End Sub


    Private Sub Button1Click(sender As Object, e As EventArgs) Handles TextBox1.TextChanged



    End Sub

    Private Sub Label1_Click(sender As Object, e As EventArgs) Handles Label1.Click

    End Sub
End Class
